docs/src/components/Layout.tsx

```diff
@@ -1,12 +1,27 @@
 import { MDXProvider } from '@mdx-js/react';
 import type { PropsWithChildren } from 'react';
 import { memo } from 'react';
+import Head from 'next/head';
 
-export const Layout = memo<PropsWithChildren<{}>>(({ children }) => {
+interface LayoutProps {
+  meta?: {
+    fileDestination?: string;
+    releaseTag?: string;
+    title?: string;
+  };
+}
+
+export const Layout = memo<PropsWithChildren<LayoutProps>>(({ meta, children }) => {
```

Comment:
I didn't know that generic type `PropsWithChildren`, thanks it would be helpful in the future :+1: